### PR TITLE
Add test for defaults.ApplyFeatureGates

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
Add test for defaults.ApplyFeatureGates

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/scheduler/algorithmprovider/defaults/defaults_test.go [hunk 1]
```diff
@@ -85,3 +87,13 @@ func TestDefaultPredicates(t *testing.T) {
 		t.Errorf("expected %v got %v", expected, result)
 	}
 }
+
+func TestApplyFeatureGates_EnabledResourceLimitsPriorityFunction(t *testing.T) {
+	utilfeature.DefaultFeatureGate.SetFromMap(map[string]bool{string(features.ResourceLimitsPriorityFunction): true})
+	ApplyFeatureGates()
```
Comment (k82cn) [inline pkg/scheduler/algorithmprovider/defaults/defaults_test.go]: What are we going to check in this test? It seems always passed :)

Comment (wojtek-t) [reply pkg/scheduler/algorithmprovider/defaults/defaults_test.go]: +1 - this test seems useless to me.

**Pull request closed without merging**